playwright/globalSetup.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  createRepositoriesManager,
+  type RepositoryConfig,
+} from "@prismicio/e2e-tests-utils";
+
+import {
+  auth,
+  baseUrl,
+  manageV2Config,
+  prismicCluster,
+} from "./playwright.config";
+
+import { setRepositoryEnvVar } from "./fixtures";
+
+async function globalSetup() {
```

Comment:
This was taken/inspired from https://github.com/prismicio/editor/blob/main/packages/wroom/e2e/globalSetup.ts